assets/src/components/editing/editor/Editor.tsx

```diff
@@ -35,6 +35,7 @@ export type EditorProps = {
   placeholder?: string;
```

Comment:
Getting the sub-editor that only worked on this special kind of table was a little tricky to get working. The stuff in here lets us override the configuration of the editor object.